packages/ui/src/components/PanelInstance.svelte

```diff
@@ -48,6 +49,10 @@
   let props: PanelProps | undefined
   function _close () {
     closePanel()
+    const lastTab = sessionStorage.getItem('last_mode')
```

Comment:
Let's remember state in panel itself, ModeSelector should not know about it. So Panel will remember state before opened and restore to it after it will be closed.

Reply:
Moved some logic to PanelInstance